server/driver_tidb.go

```diff
@@ -306,6 +307,10 @@ func (trs *tidbResultSet) Columns() ([]*ColumnInfo, error) {
 	}
 	var columns []*ColumnInfo
 	for _, v := range fields {
+		if v.ColumnAsName.O == "\\N" {
+			v.ColumnAsName = model.NewCIStr("NULL")
+		}
```

Comment:
@choleraehyq You are right,    doing this work in driver_tidb.go is not good .   I have changed it.